docs/tools/CLI/cli-device-management.md

```diff
@@ -44,8 +45,8 @@ This command asks for information about your update certificate. After completin
    - The path of the update certificate and private key.
 - Device Management developer credentials in `mbed_cloud_dev_credentials.c`
 
-<span class="notes">**Note:** The certificate created in `mbed dm init` is not suitable for production. Use it for testing and development only. To create a certificate for production purposes, use an air-gapped computer or a Hardware Security Module. When going to production, conduct a security review on your manifest signing infrastructure because it is the core of the security guarantees for update client.</span>
-
+<span class="notes">**Note:** The certificate created in `mbed dm init` is not suitable for production. Use it for testing and development only. To create a certificate for production purposes, use an air-gapped computer or a Hardware Security Module. When going to production, conduct a security review on your manifest signing infrastructure because it is the core of the security guarantees for Update client.</span>
+<!--- All of this is on the DM site. Do we link out?
```

Comment:
I would think so. @iriark01 , what are your thoughts?

Reply:
No harm in pointing people at more info, but don't eliminate the paragraph - they may not care and this info is enough.